getstarted.rst

```diff
@@ -5,73 +5,89 @@ Requirements
 ------------
 
 * `Python 3.4 or greater <https://www.python.org/download/>`_
-* `pip <https://pip.pypa.io/en/latest/installing.html>`_
-* virtualenv - ``pip install virtualenv``
+* `git <http://git-scm.com/download/mac>`_
 
 
 Installation
 ------------
 
-We will be installing nio to a virtual environment. This keeps nio and all its dependencies isolated from other python projects and environements you may already have on your machine. You can read more about them at `here <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_.
+We will be installing nio to a virtual environment. This keeps nio and all its dependencies isolated from other python projects and environements you may already have on your machine. You can read more about them `here <https://robinwinslow.co.uk/2013/12/26/python-3-4-virtual-environment/>`_.
+
+First, open a terminal in to the home directory. As a standard we will create a python virtual environement for each nio version.
+
+.. code-block:: bash
+
+    cd
+    mkdir nio
+    cd nio
+    mkdir projects
+    mkdir versions
+    cd versions
+    pyvenv-3.4 1.5.1
+
+Note: that last line will fail with Ubuntu 14.04 but can be fixed with `this <http://askubuntu.com/a/528625>`.
```

Comment:
that link doesn't really resolve the issue. It is just a workaround for a *particular virtual env*

In other words, while that link will work once, you will have to do it again in order to make another venv.

Since venv is broken on Ubuntu 14.04 (how the heck did they manage that? haha), I think we should recommend using virtualenv. 

```
sudo easy_install-3.4 pip
sudo pip3 install virtualenv
```